### DISCUSSION TITLE
[gatsby-starter-minimal-blog]: Cant find the "Lupin" title

### DISCUSSION THREAD

**Discussion by rohansingh20** [LekoArts/gatsby-themes#628] [Questions] [answered]
I have not been able to find where you contain the "Theme by LekoArts" text in the gatsby-starter-minimal-blog, could someone help?

**Comment by markgarg** [comment, marked as answer]
If you are talking about the title on the top of the page, that's in https://github.com/LekoArts/gatsby-themes/blob/master/themes/gatsby-theme-minimal-blog/src/components/header-title.tsx. However, I guess all you are looking for is adding a `siteTitle` to the `siteMetadata` in `gatsby-config.js`. 

If you are talking about the footer, it's here: https://github.com/LekoArts/gatsby-themes/blob/master/themes/gatsby-theme-minimal-blog/src/components/footer.tsx
You'll need to shadow it in your code if you'd like to override it.

**Reply by markgarg** [reply]
You can find the information here: https://github.com/LekoArts/gatsby-themes/tree/master/themes/gatsby-theme-minimal-blog#additional-configuration
